portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,138 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f5f6fa;
+  margin: 30px 200px 30px 200px;
+}
+
+div {
+  color: darkslategrey;
+}
+
+.bar {
+  background-color: #f48b52;
+  padding: 20px;
+}
+
+#bottom-bar {
+  text-align: -webkit-center;
+}
+
+#name-container {
+  color: darkslategrey;
+  font-size: 20px;
+}
+
+#navbar-container {
+  float: right;
+  font-size: 20px
+}
+
+#content-container {
+  background-color: #b1dfd5;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  padding: 20px;
+  text-align: -webkit-center;
+}
+
+a {
+  color: #f5f6fa;
+  margin: 10px;
+  text-decoration: none;
+}
+
+a.repo {
+  color: #f48b52;
+}
+
+.active {
+  color: darkslategrey;
+}
+
+a:hover {
+  color: darkslategrey;
+}
+
+a.repo:hover {
+  color: #f5f6fa;
+}
+
+#profile-picture {
+  width: 300px;
+}
+
+button {
+  background-color: #b1dfd5;
+  border: none;
+  color: darkslategrey;
+  cursor: pointer;
+  font-family: "Times New Roman";
+  font-size: inherit;
+  font-weight: inherit;
+  padding: 0px;
+  text-decoration: underline;
+}
+
+button:hover {
+  color: #f48b52;
 }
 
 #greeting-container {
-  margin-top: 20px;
+  text-decoration: none;
+}
+
+#quote-container {
+  color: #f48b52;
+  font-size: 18px;
+  margin-top: 10px;
+}
+
+.row {
+  display: -ms-flexbox;
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 15px;
+  padding: 0 4px;
+  -ms-flex-wrap: wrap;
+}
+
+.column {
+  flex: 33%;
+  max-width: 33%;
+  padding: 0 4px;
+  -ms-flex: 33%;
+}
+
+.column img {
+  margin-top: 8px;
+  vertical-align: middle;
+  width: 100%;
+}
+
+.info-container {
+  margin-right: 20px;
+  margin-left: 20px;
+}
+
+.info {
+  margin: 10px;
+  padding: 10px;
+}
+
+.project-name {
+  font-size: 20px;
```

Comment:
FYI good reading about px vs pt vs em for font sizes:
https://css-tricks.com/css-font-size/